com.unity.multiplayer.mlapi/Runtime/SceneManagement/NetworkSceneManager.cs

```diff
@@ -335,93 +341,49 @@ private static void OnSceneUnloadServer(Guid switchSceneGuid)
 
         private static void OnSceneUnloadClient(Guid switchSceneGuid, Stream objectStream)
         {
-            if (!NetworkManager.Singleton.NetworkConfig.EnableSceneManagement || NetworkManager.Singleton.NetworkConfig.UsePrefabSync)
+            var networkObjects = UnityEngine.Object.FindObjectsOfType<NetworkObject>();
+            NetworkManager.Singleton.SpawnManager.ClientCollectSoftSyncSceneObjectSweep(networkObjects);
+
+            using (var reader = PooledNetworkReader.Get(objectStream))
             {
-                NetworkManager.Singleton.SpawnManager.DestroySceneObjects();
```

Comment:
This is not needed within the NetworkSceneManager since UsePrefabSync no longer exists and this code should never get reached if EnableSceneManagement is disabled.